internal/pkg/config/output.go

```diff
@@ -18,8 +18,11 @@ import (
 	"time"
 
 	urlutil "github.com/elastic/elastic-agent-libs/kibana"
+	"github.com/elastic/elastic-agent-libs/logp"
 	"github.com/elastic/elastic-agent-libs/transport/httpcommon"
 	"github.com/elastic/elastic-agent-libs/transport/tlscommon"
+
+	"github.com/elastic/fleet-server/v7/internal/pkg/logger"
```

Comment:
I think this wrapper specifically is causing some cyclical imports; it probably needs to be refactored

Reply:
Done.  Please take a look.  Thanks.